osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyCalculator.cs

```diff
@@ -25,7 +26,7 @@ public class TaikoDifficultyCalculator : DifficultyCalculator
         private const double colour_skill_multiplier = 0.375 * difficulty_multiplier;
         private const double stamina_skill_multiplier = 0.375 * difficulty_multiplier;
 
-        public override int Version => 20241007;
+        public override int Version => 20241115;
```

Comment:
Please don't change this. We'll do this when preparing for a deploy to make sure we're not bumping versions multiple times.